server/routes/user.js

```diff
@@ -28,7 +28,31 @@ router.patch('/user/userUpdate', authenticate, (req,res) => {
 });
 
 router.post('/user/logout', authenticate, (req,res) => {
-    userLibs.logOut(req).then((success) => {
+    let token = req.header('x-authentication');
+    userLibs.logOut(token).then((success) => {
+        res.status(success.status).send(success);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    })
+})
+
+router.delete('/admin/deactivate/:uuid', authenticate, (req,res) => {
```

Comment:
why using admin??

Reply:
Because this is a route for admin where only admin can deactivate the user

Reply:
its a user route make it user not admin

Reply:
Done